osu.Game.Rulesets.Catch/Beatmaps/CatchBeatmapProcessor.cs

```diff
@@ -209,7 +209,7 @@ private static void initialiseHyperDash(IBeatmap beatmap)
                 }
             }
 
-            palpableObjects.Sort((h1, h2) => h1.StartTime.CompareTo(h2.StartTime));
+            palpableObjects = palpableObjects.OrderBy(h => h.StartTime).ToList();
```

Comment:
If the stability of the sort is important, then there should be an inline comment about it or a test covering it. Or both. Otherwise there's no transparent reason why this particular one way to sort out of the multiple ones offered by the standard library is used here precisely.